<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSTOTestForm</class>
 <widget class="QDialog" name="SSTOTestForm">
  <property name="windowTitle">
   <string>Тестирование оборудования ССТО</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelStationName">
     <property name="text"><string>Станция / корабль:</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="stationName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTestType">
     <property name="text"><string>Тип теста:</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="testType">
     <item><property name="text"><string>GMDSS A1</string></property></item>
     <item><property name="text"><string>GMDSS A2</string></property></item>
     <item><property name="text"><string>Инмарсат</string></property></item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTestDate">
     <property name="text"><string>Дата теста:</string></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="testDate"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelResult">
     <property name="text"><string>Результат:</string></property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="testResult"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonSubmit">
     <property name="text"><string>Сохранить</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>